document-generator-company-report/src/main/java/uk/gov/companieshouse/document/generator/company/report/mapping/mappers/CompanyReportMapperDecorator.java

```diff
@@ -50,16 +56,31 @@ public CompanyReport mapCompanyReport(CompanyReportApiData companyReportApiData)
                 companyReport.setPreviousNames(setPreviousNames(companyReportApiData.getCompanyProfileApi().getPreviousCompanyNames()));
             }
 
-            companyReport.setCurrentAppointments(setCurrentAppointments(companyReportApiData.getOfficersApi()));
+            if(companyReportApiData.getOfficersApi() != null) {
+                if (companyReportApiData.getOfficersApi().getItems().size() > 0) {
```

Comment:
instead of the 2 if statements can just be 
`if(companyReportApiData.getOfficersApi() != null && companyReportApiData.getOfficersApi().getItems().size() > 0)`